src/presentational/CheckboxGroup.jsx

```diff
@@ -0,0 +1,65 @@
+import { Component, PropTypes } from 'react';
+
+export default class CheckboxGroup extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      checked: null,
+    }
+    this.getCheckboxValues = this.getCheckboxValues.bind(this);
+    this.handleChange = this.handleChange.bind(this);
+  }
+
+  getCheckboxValues(name) {
+    const values = [];
+    const options = document.getElementsByName(name);
+    for (var i = 0; i < options.length; i++) {
+      if (options[i].checked) {
+        values.push(options[i].value);
+      }
+    }
+    return values;
+  }
+
+  handleChange(e) {
+    e.target.value = this.getCheckboxValues(e.target.name);
+    this.props.onChange(e);
+    this.setState({checked: e.target.checked});
```

Comment:
Does `state.checked` ever get used? I can't see it anywhere else in this component.